app/testOnly/LoginRequestMaker.scala

```diff
@@ -92,4 +93,15 @@ object LoginRequestMaker {
     "state" -> irSaEnrolment.enrolmentStatus.toString
   )
 
+  private def makeIrSaAndMtdItEnrolmentJson(mtdItEnrolment: MtdItEnrolment): JsObject = Json.obj(
+    "key" -> "HMRC-MTD-IT",
+    "identifiers" -> Json.arr(
+      Json.obj(
+        "key" -> "MTDITID",
+        "value" -> mtdItEnrolment.nino.value
```

Comment:
might not make too much sense to have `nino` in the `MtdItEnrolment` case class anymore - it'd make sense to have a separate `MtdItId` field on the test-only form which when left blank will just result in a random id being generated. Not sure what the form of the id is but if we generate 8 random digits or something then that should be good enough